lib/pubsub/index.js

```diff
@@ -199,16 +199,10 @@ PubSub.prototype.getTopics = function(query, callback) {
  *
  * @example
  * pubsub.createTopic('my-new-topic', function(err, topic, apiResponse) {
- *   topic.publish({
- *     data: 'New message!'
- *   }, function(err) {});
```

Comment:
Publishing a message to a topic you created will likely have no subscriptions registered yet, which results in [the message being lost](https://github.com/GoogleCloudPlatform/gcloud-node/issues/696#issuecomment-116835719). I figured it's better to not even suggest that.